pkg/scheduler/factory/factory_test.go

```diff
@@ -245,58 +239,6 @@ func PriorityTwo(pod *v1.Pod, nodeNameToInfo map[string]*schedulernodeinfo.NodeI
 	return []schedulerapi.HostPriority{}, nil
 }
 
-func TestDefaultErrorFunc(t *testing.T) {
```

Comment:
I believe we should keep the test and change the FIFO queue to a priority queue.

Reply:
Updated. To compare the testPod with the one stored in the priority queue, I added a public function in scheduling_queue.go to get and compare the pod in backoff queue or unschedulable queue.